read-fonts/src/array.rs

```diff
@@ -142,6 +142,13 @@ impl<'a, T: FontRead<'a> + VarSize> VarLenArray<'a, T> {
             }
 
             let item_len = T::read_len_at(data, 0)?;
+            // If the length is 0 then then it's not useful to continue
```

Comment:
this maybe wants to be in a separate PR with a test case?

Reply:
Oops, I meant to do this but forgot. Will submit a separate PR.